gobblin-kubernetes/gobblin-service/base-cluster/deployment.yaml

```diff
@@ -57,7 +59,12 @@ spec:
             claimName: shared-jobs-claim
       containers:
         - name: gobblin-standalone
-          image: will97/gobblin-standalone:latest
+          image: apache/gobblin:latest
+          args: ["--mode", "standalone", "--jvmopts", "-XX:+UnlockExperimentalVMOptions -XX:+UseCGroupMemoryLimitForHeap"]
           volumeMounts:
             - name: shared-jobs
               mountPath: /tmp/gobblin-standalone/jobs
+          env:
+            - name: GOBBLIN_JOB_CONFIG_DIR
+              value: /tmp/gobblin-standalone/jobs
```

Comment:
/tmp/ is not a good place to store configuration, as it can get cleaned up at any moment - https://serverfault.com/questions/377348/when-does-tmp-get-cleared

If those configs rarely change while container is running, you can put them in /etc/gobblin . For files that are constantly changing , use /var/... 

See https://en.wikipedia.org/wiki/Filesystem_Hierarchy_Standard for explanation of linux file system structure

Reply:
Definitely learned something new today, will make the change. Thanks!